src/validate.js

```diff
@@ -32,6 +32,18 @@ const checkHeadingSpaces = (text, level) => {
     return regex.test(text);
 };
 
+/**
+ * Checks if the given version is not released yet
+ *
+ * @param {string} version
+ * @returns {boolean}
+ */
+const isPrerelease = (version) => {
+    const lowerCasedVersion = version.toLowerCase();
+    const preReleases = ['alpha', 'rc', 'beta', 'dev'];
```

Comment:
what is `dev`?

Reply:
We can remove it not needed, I wanted to cover most of the examples of semver.
<img width="591" alt="Screenshot 2021-05-12 at 19 25 35" src="https://user-images.githubusercontent.com/1938043/118018239-d6ab2880-b357-11eb-9af2-8cf70b096acc.png">

Reply:
where did you get these examples?

Reply:
From the official page of [semver](https://semver.org/)

Reply:
Interesting, I always thought it's limited to alpha, beta, rc. TIL

Reply:
resolved